spring-cloud-context/src/main/java/org/springframework/cloud/context/named/NamedContextFactory.java

```diff
@@ -67,6 +67,12 @@ public void setConfigurations(List<C> configurations) {
 		return new HashSet<>(contexts.keySet());
 	}
 
+	protected void createAndCacheContexts() {
```

Comment:
I would create an issue in s-c-commons for this that would reference the one in netflix.